### DISCUSSION TITLE
Why am i not on the list

### DISCUSSION THREAD

**Discussion by hakdiy** [IBAX-io/go-ibax#1845] [Rewards]
![lcbxx](https://user-images.githubusercontent.com/96067508/152445254-482055e0-ca13-442b-bedc-2e0bf3c6fe83.png)
Why am i not on the list. Could you help

**Comment by yolandadadada** [comment]
What's on the list, can you describe it in detail?

**Comment by hakdiy** [comment]
Hi My mission score is below but I'm not on your airdrop list. Can you help me? twitter  @hakdiy

   8 Şubat 2022 Salı 09:40:58 GMT+3 tarihinde, yolandadada ***@***.***>şunu yazdı:  
 
 


What's on the list, can you describe it in detail?

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by yolandadadada** [reply]
What airdrop list? We don't have an airdrop list.

**Comment by hakdiy** [comment]
hi what is this
    10 Şubat 2022 Perşembe 13:15:33 GMT+3 tarihinde, yolandadada ***@***.***>şunu yazdı:  
 
 


What airdrop list? We don't have an airdrop list.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
